contents/blog/collaborating-on-posthog.md

```diff
@@ -0,0 +1,86 @@
+
+To address the above we introduced robust permalinks to insights that can easily be shared across multiple mediums. These links can now be used to link to a specific insight (with all configurations), even if you have multiple projects.
+
+
+![Concept of how insight links changed to permalinks today](../images/blog/collaborating-on-posthog_1.png)
+
+
+Dashboards can easily be shared too, and even if you share a link to a dashboard in a different project, if the user has access to that project, they'll be automatically switched to that project.
+
+For even larger teams, new challenges arise when collaborating with dashboards. In particular, we found a strong need to prevent accidental changes that would affect metrics for others. This was mentioned frequently for larger organizations. In small teams this is usually not problematic as it's common for a single person to own the analytics process. So we introduced granular permission controls to dashboards to provide confidence to users that dashboards haven't changed when they shouldn't.
+
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-dashboard-permissions.png" width="500" alt="Screenshot of dashboard permissions" />
+
+
+### Use case two: Discovering interesting stuff you weren't looking for
+Sometimes you don't know what you're looking for, or you don't know where to start. There's many different avenues to improving your product, and in particular there might be opportunities for improvements across a myriad of places. For instance, we recently [discovered](https://github.com/PostHog/posthog/pull/7973) users seldom use our paths advanced features and these could introduce mental burden to users.
+
+We created a project homepage with the goal of surfacing helpful insights very quickly that you may not have had in your radar. These insights can hopefully help you find product avenues worth improving. By making it the default page on PostHog, our aim is to help users make quick discoveries. We'll surface your most important product & company metrics here as well as popular insights. We're leveraging social proof from what other teammates are doing in your app to help you discover more product improvements.
+
+In addition, if your team uses the recordings feature we'll also surface the most recent recordings here. We found that on average returning users to the recordings feature discover proportionally more than 3X than other users (more context [here](https://github.com/PostHog/posthog/issues/8595#issuecomment-1056916848)). This should also help drive qualitative discoveries about your product.
+
+
+Mockup / screenshot here.
```

Comment:
Reminder to add this

Reply:
@clarkus @rcmarron which mockup or screenshot should we use for the project home page?

Reply:
Defer to @rcmarron but all the designs are at https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=7877%3A87159. The MVP - populated screen is probably the closest to what a user would see. Another idea could be to screenshot this working on our demo project?